### PR TITLE
update and link to CLI > hydrate / package

### LINKED ISSUE #488: .arc-env mentioned only once, not documented
**Describe the issue**
According to https://arc.codes/docs/en/reference/cli/sandbox:

> `ARC_LOCAL`- If present and used in conjunction with `NODE_ENV=staging|production`, emulates live staging or production environment
> ...
>
> * Uses your local `.arc-env` file’s `@staging` or `@production` environment variables
>
> ...

This is the only mention that I can find of the `.arc-env` file. I see no other documentation about the `.arc-env` file, and no explanation of its format or contents. I suspect that this is actually supposed to refer to `preferences.arc`.

**Steps to reproduce**
N/A, this is a documentation bug.

**Expected behavior**
N/A, this is a documentation bug.

**Screenshots**
N/A, this is a documentation bug.

### PR DESCRIPTION
**Pull request by tbeseda**
This started as quick change to remove the single reference to `.arc-env`, ~~but I ended up reintroducing docs for the hydrate and package commands with some updates~~.

After discussion, this removes the `package` doc and updates `hydrate`.

also minor updates to other docs and fix style issue for narrow documents

fixes #488

### CODE REVIEW

**Comment by ryanblock** [comment]
Hmm, I'd limit how deep you go on those two! As Architect has evolved Hydrate and Package specifically have become incredibly important libraries whose command usage have become a bit more internal / diagnostic.

For example, Hydrate, which in the early days was something you might run all the time by hand, is now largely papered over and automated thanks to Lambda tree shaking.

Package, which used to be a fast way to produce your application's CloudFormation doc, no longer reflects the deployed application, due to mutations within plugins (and legacy macros), or backwards compatibility checks, etc.

Anyway, for whatever all that is worth!

**Comment by tbeseda** [comment]
> As Architect has evolved Hydrate and Package specifically have become incredibly important libraries whose command usage have become a bit more internal / diagnostic.

I thought about this, too. And I figured that's why these docs weren't linked in the sidebar. On a whim I decided to add them back (in an abbreviated way) since I'd seen them discussed in support channels recently.

> For example, Hydrate, which in the early days was something you might run all the time by hand

Definitely not needed as often. I very rarely run it by hand, but it can be useful to make sure tests are ready to run.

> Package, which used to be a fast way to produce your application's CloudFormation doc, no longer reflects the deployed application, due to mutations within plugins (and legacy macros), or backwards compatibility checks, etc.

This is true. I'm not sure there's much of a use case for this command anymore. Especially with Arc 10 projects likely using way more plugins. I agree, this one can be removed.

**Comment by ryanblock** [comment]
Now that we are talking about it, I’d actually propose we remove `arc package` as a CLI command in Arc 10: @brianleroux @filmaj wdyt?

**Comment by brianleroux** [comment]
err, no! at least not the capability; we promise full CloudFormation compat and eject to SAM and want to keep that promise moving forward. we should document the quirks.

File: src/views/docs/en/reference/cli/hydrate.md [hunk 1]
```diff
@@ -1,77 +1,40 @@
...
 category: CLI
-description: Quickly install and update dependencies for all functions in the src directory.
+description: Install and update dependencies for all functions in a project.
 ---
 
-## Overview
+Ensure that all functions managed by Architect have their dependencies installed. Functions containing all its required dependencies are considered to be "hydrated" - thus the name!
 
-[`@architect/hydrate`](Architect env) ensures that all functions managed by architect have their dependencies installed. Functions containing all its required dependencies are considered to be 'hydrated' - thus the name!
+Importantly, `arc hydrate` will also copy shared code (from `src/shared` and `src/views`) into all functions.
```
Comment (macdonst) [inline src/views/docs/en/reference/cli/hydrate.md:9]: `arc hydrate` only copies `src/views` into functions that are GET routes right?

**Review by macdonst — COMMENTED** [review]
Just one clarifying question.

File: src/views/docs/en/reference/cli/hydrate.md (hunk 1 above)
Comment (tbeseda) [reply src/views/docs/en/reference/cli/hydrate.md:9]: Good point! I will update. ty

**Comment by tbeseda** [comment]
My proposal is move forward withe `hydrate` doc linked in the sidebar and the `package` doc like but un-linked/indexed.

If the package command is removed, we delete the doc. If package gains more usage or is mentioned in other docs, we can reference it and possibly add to the sidebar.

File: src/views/docs/en/reference/cli/hydrate.md (hunk 1 above)
Comment (ryanblock) [reply src/views/docs/en/reference/cli/hydrate.md]: Yes, and also:
```
@http
get /
get /foo
get /lol
post /bar
patch /fiz

@views
get /foo
post /bar
patch /fiz
```

In the above scenario, of the 5 routes listed, only the 3 under `@views` get `src/views`.

**Pull request merged**